src/Seller/ShippingV2/Dto/GetTrackingResult.php

```diff
@@ -25,9 +25,9 @@ final class GetTrackingResult extends Dto
      */
     public function __construct(
         public string $trackingId,
-        public string $alternateLegTrackingId,
-        public array $eventHistory,
-        public \DateTimeInterface $promisedDeliveryDate,
-        public TrackingSummary $summary,
+        public ?string $alternateLegTrackingId = null,
+        public array $eventHistory = [],
+        public ?\DateTimeInterface $promisedDeliveryDate = null,
+        public ?TrackingSummary $summary = null,
```

Comment:
The sandbox was not replying back `alternateLegTrackingId`, causing tests to fail